srcpkgs/OpenJK/template

```diff
@@ -0,0 +1,53 @@
+# Template file for 'OpenJK'
+pkgname=OpenJK
+version=0.0.20210922
+revision=1
+_gitrev=43e9a3da555a487e23081bbd03101e7c926a1e83
+wrksrc="${pkgname}-${_gitrev}"
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr/lib/${pkgname} -DBuildJK2SPEngine=on
+ -DBuildJK2SPGame=on -DBuildJK2SPRdVanilla=on"
+hostmakedepends="pkg-config"
+makedepends="libjpeg-turbo-devel libpng-devel zlib-devel SDL2-devel"
+short_desc="Open Source Implementation of Jedi Knight II and III Engine"
+maintainer="Kenneth Dodrill <kmdodrill@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/JACoders/OpenJK"
+distfiles="https://github.com/JACoders/OpenJK/archive/${_gitrev}.tar.gz"
+checksum=efd2e3d96fe449819c4276dfea7ae7d591591ee0e651167844110e073f2edb3c
+
+post_install() {
+	OJ_ARCH=$(uname -m)
+	case "$OJ_ARCH" in
+		arm*) OJ_ARCH=arm;;
+		i*86*) OJ_ARCH=i386;;
+	esac
```

Comment:
no, that's not how this works

it should be the *script* doing it, not the template

and at that point you can just make them files in files/ and copy them over